addons/web/static/src/core/notifications/notification.xml

```diff
@@ -2,14 +2,16 @@
 <templates xml:space="preserve">
 
     <t t-name="web.NotificationWowl" owl="1">
-        <div t-attf-class="o_notification {{className}}"
+        <div t-attf-class="o_notification toast show border-{{props.type}} overflow-hidden mb-2"
         role="alert" aria-live="assertive" aria-atomic="true">
-            <div class="o_notification_body">
-                <button type="button" class="close o_notification_close"
+            <div t-attf-class="toast-header pb-1">
+                <div t-if="props.title" t-attf-class="o_notification_title font-weight-bold text-{{props.type}} mr-auto" t-esc="props.title"/>
```

Comment:
When there is no title, close button is on the left. See "Connection lost. Trying to reconnect..." notification.

Reply:
Fixing it to be always on the right in case of no title. 

But I think, it would be great to have a title & body for every notifications (mandatory) 
In this case, maybe - Title: Connection lost  / body content: Trying to reconnect ...
Is this something faisible?

Reply:
Or you could remove "t-if" on this node. The div will be present but not the content.

Reply:
About that node, why using `<div class="font-weight-bold">` instead of `<strong>` as even the Bootstrap example use it and it correctly puts the emphasis on the title?